### PR TITLE
Commonjs support

### PR DESCRIPTION
**Pull request by ro0gr**
Hey @san650 👋 

I'm currently playing around the https://github.com/san650/ember-cli-page-object/issues/512, and noticed that `Ceibo` can't be imported in the node environment. So this change brings an ability to use it under Node.js.

Also, in order to have a commonjs test, I've upgraded test deps, and upgraded the Node.js version from 4.5 to 10. I think technically I could leave 4.5, and live w/o object desctrution, but please, not w/o object desctrution! 😆

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+  npm: true
+
+branches:
+  only:
+    - master
+
+jobs:
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-canary
```
Comment (ro0gr) [inline .travis.yml]: Oops

Comment (san650) [inline .travis.yml]: I think we can remove this line since we don't use ember in this repo.

**Review by san650 — APPROVED** [review]
Wow nice update to the library 👏 

Left a nit comment, let me know if you want to update the PR or if you want to merge it as-is.

File: .travis.yml (hunk 1 above)
Comment (san650) [reply .travis.yml]: just commented about the same

**Comment by ro0gr** [comment]
honestly, I'm not sure if the travis config is correct, I'm looking for the build to finish https://travis-ci.org/github/san650/ceibo/builds/758277944

if it'd be fine, I'm 👌  to merge

**Comment by san650** [comment]
@ro0gr I've run the tests locally and its working so I'll go ahead and merge the PR.

**Comment by san650** [comment]
released v2.1 with this change https://www.npmjs.com/package/ceibo

Thanks @ro0gr !!

**Comment by ro0gr** [comment]
Thank for quick feedback @san650!

**Pull request merged**